piety-promoted-v3/modernized/17-appendix.adoc

```diff
@@ -220,7 +228,7 @@ in the first year of her travels in England,
 is particularly thought worthy of preservation.
 She seems, previously,
 to have been tried by means of much deprivation of that heavenly comfort and support,
-which she loved and sought.`'
+which she loved and sought.
 
 As I quietly rode along, '`says she,
```

Comment:
I wasn't sure if this should be embedded with the "says she" part in it.

Reply:
I think it's fine how you left it. 👍